web/components/layouts/Main.tsx

```diff
@@ -21,6 +21,12 @@ import { ServerRenderedHydration } from '../ServerRendered/ServerRenderedHydrati
 
 import { Theme } from '../theme/Theme';
 
+const globalStyle = `
```

Comment:
Since this is a static style, is there any way to put it in a CSS module like all the rest of the styles?  That `customStyles` is different because it comes from the API.

Reply:
This might be a bit tricky. A CSS module is not allowing global selectors. And if I try to add just some styles via `Main.scss` NextJS is complaining about it that I can't put global styles there.

I can take a closer look at it tmrw maybe I'll find something 🙂

Reply:
I've added the `useLockBodyScroll` hook from `react-use` to it. It works.

After reading through some articles and stackoverflow, this seemed to be an appropriate solution. Since NextJS and CSS Modules really don't want to apply global styles in a module. Everybody was pointing either to the global Style solution in the head or using:

https://nextjs.org/blog/styling-next-with-styled-jsx